maint_tools/test_docstrings.py

```diff
@@ -17,7 +17,6 @@
     "LabelSpreading",
     "LocalOutlierFactor",
     "LocallyLinearEmbedding",
-    "MeanShift",
```

Comment:
It seems that you kept the commit from your MeanShift branch instead of branching from `main`. Could you revert and only have the commit specifically for the estimator that you are changing.

Reply:
I see, Since I worked on both of them simultaneously I may have mixed it up, I will redo the changes in a new branch, and raise a new PR